WcaOnRails/app/views/competitions/_no_competitions_found.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="col-md-12">
+  <%= alert :warning do
+    unless params[:event_ids].empty?
```

Comment:
nit: Personally, I try to avoid unless and would write this as an `if` and swap the two bodies.